src/dashboard/cd-dashboard.vue

```diff
@@ -54,8 +57,16 @@ export default {
 
     &__right-column {
       background-color: #f4f5f6;
-      max-width: 340px;
       flex: 1;
     }
   }
+
+  @media (max-width: @screen-xs-max) {
+    .cd-dashboard {
+
+      &__container {
+        flex-direction: column;
```

Comment:
Shouldn't that be column-reverse ?

Reply:
YES and i can delete the order stuff too nice